docs/proposals/tool-to-backfill-tsdb-blocks.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Tools to backfill TSDB blocks"
+linkTitle:
+weight: 1
+slug:
+---
+
+- Author: [Siddharth Asthana](https://github.com/edith007)
+- Date: March 2023
+- Status: draft
+
+## Overview
+
+There might be a lot of cases where the users might already be using Prometheus and later decides to switch to Cortex. Cortex will take care of the metrics which will be generated after the switch, but at present there is no way to backfill the old Prometheus TSDB blocks to Cortex. This proposal is to add the ability in cortex to fetch the Prometheus TSDB blocks and backfill the data.
```

Comment:
Is Thanos blocks part of the goal of the proposal?

Reply:
yeah.. i think it can be `prometheus/thanos`?

Reply:
I would support cortex and mimir blocks too. But let's make priorities:
1. prometheus (Which version?)
2. thanos (Which version?)
3. cortex (Which version?)
4. mimir (Which version?)